Library/Homebrew/diagnostic.rb

```diff
@@ -513,7 +514,7 @@ def check_user_path_3
           Homebrew's sbin was not found in your PATH but you have installed
           formulae that put executables in #{HOMEBREW_PREFIX}/sbin.
           Consider setting the PATH for example like so
-            echo 'export PATH="#{HOMEBREW_PREFIX}/sbin:$PATH"' >> #{shell_profile}
+            echo 'export PATH="#{HOMEBREW_PREFIX}/sbin:$PATH"' >> #{Utils::Shell.shell_profile}
```

Comment:
Considering we start to support shell like fish, the `echo 'export ...` part have to handle in a shell by shell case.